client/my-sites/checkout/composite-checkout/wpcom/components/wp-checkout-order-summary.js

```diff
@@ -102,6 +100,88 @@ export default function WPCheckoutOrderSummary() {
 	);
 }
 
+function LoadingCheckoutSummaryFeaturesList() {
+	return (
+		<>
+			<LoadingCopy />
+			<LoadingCopy />
+			<LoadingCopy />
+		</>
+	);
+}
+
+function CheckoutSummaryFeaturesList() {
+	const hasDomainsInCart = useHasDomainsInCart();
+	const domains = useDomainsInCart();
+	const hasPlanInCart = useHasPlanInCart();
+	const translate = useTranslate();
+
+	const supportText = hasPlanInCart
+		? translate( 'Email and live chat support' )
+		: translate( 'Email support' );
+
+	let refundText = translate( 'Money back guarantee' );
+	if ( hasDomainsInCart && ! hasPlanInCart ) {
+		refundText = translate( '4 day money back guarantee' );
+	} else if ( hasPlanInCart && ! hasDomainsInCart ) {
+		refundText = translate( '30 day money back guarantee' );
+	}
+
+	return (
+		<CheckoutSummaryFeaturesListUI>
+			{ hasDomainsInCart &&
+				domains.map( ( domain ) => {
+					return <CheckoutSummaryFeaturesListDomainItem domain={ domain } key={ domain.id } />;
+				} ) }
+			<CheckoutSummaryFeaturesListItem>
+				<WPCheckoutCheckIcon />
+				{ supportText }
+			</CheckoutSummaryFeaturesListItem>
+			<CheckoutSummaryFeaturesListItem>
+				<WPCheckoutCheckIcon />
+				{ refundText }
+			</CheckoutSummaryFeaturesListItem>
+		</CheckoutSummaryFeaturesListUI>
+	);
+}
+
+function CheckoutSummaryFeaturesListDomainItem( { domain } ) {
+	const translate = useTranslate();
+	return (
+		<CheckoutSummaryFeaturesListItem>
+			<WPCheckoutCheckIcon />
+			{ domain.wpcom_meta.is_bundled ? (
+				translate( '{{strong}}%(domain)s{{/strong}} - %(bundled)s', {
+					components: {
+						strong: <strong />,
+					},
+					args: {
+						domain: domain.wpcom_meta.meta,
+						bundled: translate( 'free with plan' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **33** times:
`translate( 'Free with Plan' )` *ES Score: 9*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/checkout-dynamic-plan-features)